src/ZenstruckFoundryBundle.php

```diff
@@ -221,84 +225,10 @@ public function loadExtension(array $config, ContainerConfigurator $configurator
-                $container->removeDefinition('.zenstruck_foundry.maker.factory.doctrine_scalar_fields_default_properties_guesser');
-            }
-
-            $container->getDefinition('.zenstruck_foundry.maker.factory.generator')
-                ->setArgument('$forceProperties', $config['instantiator']['always_force_properties'] ?? false);
-        } else {
-            $configurator->import('../config/command_stubs.php');
-        }
-
-        if (isset($bundles['DoctrineBundle']) || isset($bundles['DoctrineMongoDBBundle'])) {
-            $configurator->import('../config/persistence.php');
-        }
-
-        if (isset($bundles['DoctrineBundle'])) {
-            $configurator->import('../config/orm.php');
-
-            $container->getDefinition('.zenstruck_foundry.persistence.database_resetter.orm.abstract')
-                ->replaceArgument('$managers', $config['orm']['reset']['entity_managers'])
-                ->replaceArgument('$connections', $config['orm']['reset']['connections'])
-            ;
-
-            /** @var ResetDatabaseMode $resetMode */
-            $resetMode = $config['orm']['reset']['mode'];
-            $container->getDefinition(OrmResetter::class)
-                ->setClass(
-                    match ($resetMode) {
-                        ResetDatabaseMode::SCHEMA => SchemaDatabaseResetter::class,
-                        ResetDatabaseMode::MIGRATE => MigrateDatabaseResetter::class,
-                    }
-                );
-
-            if (ResetDatabaseMode::MIGRATE === $resetMode) {
-                $container->getDefinition(OrmResetter::class)
-                    ->replaceArgument('$configurations', $config['orm']['reset']['migrations']['configurations'])
-                ;
-            }
-        }
-
-        if (isset($bundles['DoctrineMongoDBBundle'])) {
-            $configurator->import('../config/mongo.php');
-
-            $container->getDefinition(MongoResetter::class)
-                ->replaceArgument(0, $config['mongo']['reset']['document_managers'])
-            ;
-        }
-
-        $configurator->import('../config/in_memory.php');
-
-        $container->registerForAutoconfiguration(InMemoryRepository::class)->addTag('foundry.in_memory.repository');
-
-        if (false === $config['persistence']['flush_once']) {
-            trigger_deprecation('zenstruck/foundry', '2.5', 'Not setting "zenstruck_foundry.persistence.flush_once" to true is deprecated. This option will be forced to true in 3.0');
-        }
-
-        $container->setParameter('zenstruck_foundry.persistence.flush_once', $config['persistence']['flush_once']);
+        $this->configureMakers($configurator, $container, $config);
+        $this->configurePersistence($container, $configurator, $config);
+        $this->configureInMemory($configurator, $container);
+        $this->configureFixturesStory($configurator, $container);
```

Comment:
I think it is time to group those in methods :slightly_smiling_face: 

new code is only in the last method `configureFixturesStory()`